src/app/common/modals/about-doubtfire-modal/about-doubtfire-modal.component.ts

```diff
@@ -5,12 +5,16 @@ import { Injectable, Component, Inject, OnInit } from '@angular/core';
 import { DoubtfireConstants } from 'src/app/config/constants/doubtfire-constants';
 import { AboutDoubtfireModalService } from '../about-doubtfire-modal/about-doubtfire-modal.service';
 import { GithubProfile } from '../about-doubtfire-modal/GithubProfile';
+import {GithubProfileOthers} from '../about-doubtfire-modal/GithubProfileOthers';
 
 import { MatDialog, MAT_DIALOG_DATA } from '@angular/material';
 
 export interface AboutDialogData {
   externalName: string,
-  contributors: GithubProfile[]
+  contributors: GithubProfile[],
```

Comment:
All contributors should be in one array, then can have some inheritance if required, but this most likely isn't necessary just have them all as one type.